### PR TITLE
Update prettier toolchain

### PR DESCRIPTION
**Pull request by ooflorent**
**What kind of change does this PR introduce?**

build related change

**Did you add tests for your changes?**

no

**If relevant, link to documentation update:**

n/a

**Summary**

Bump prettier and its ESLint integration.

**Does this PR introduce a breaking change?**

no

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = {
 		"semi": "error",
 		"no-template-curly-in-string": "error",
 		"no-caller": "error",
+		"no-control-regex": "off",
```
Comment (ooflorent) [inline .eslintrc.js]: ESLint denies `\x00-\x32`. Webpack uses this range to detect binary data.

File: .eslintrc.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
 	root: true,
 	plugins: ["prettier", "node"],
-	extends: ["eslint:recommended", "plugin:node/recommended"],
+	extends: ["eslint:recommended", "plugin:node/recommended", "plugin:prettier/recommended"],
```
Comment (ooflorent) [inline .eslintrc.js]: Disable style related rules

File: package.json [hunk 3]
```diff
@@ -104,7 +105,7 @@
     "lint-files": "npm run lint && npm run schema-lint",
     "lint": "eslint lib bin hot buildin \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\"",
     "fix": "npm run lint -- --fix",
-    "pretty-files": "prettier \"lib/**.*\" \"bin/**.*\" \"hot/**.*\" \"buildin/**.*\" \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\" --write",
+    "pretty-files": "prettier \"lib/**/*.js\" \"bin/*.js\" \"hot/*.js\" \"buildin/*.js\" \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\" --write",
```
Comment (ooflorent) [inline package.json]: Masks were wrong

**Comment by sokra** [comment]
Thanks

**Pull request merged**